test/airtable_error.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var AirtableError = require('../lib/airtable_error');
+
+describe('AirtableError', function() {
+    describe('#toString', function() {
+        it('includes the provided `error` value', function() {
+            var error = new AirtableError('value of error parameter');
+            expect(error.toString()).toEqual(expect.stringContaining('value of error parameter'));
+        });
+
+        it('includes the provided `message` value', function() {
+            var error = new AirtableError(null, 'value of message parameter');
+            expect(error.toString()).toEqual(expect.stringContaining('value of message parameter'));
+        });
+
+        it('includes the provided `statusCode` value', function() {
+            var error = new AirtableError(null, null, 'value of statusCode parameter');
```

Comment:
If/when we strongly type AirtableError, statusCode would be a number so we should change this test to pass in a number instead of a string.

Reply:
Good point. Done!